src/main/scala/viper/silver/ast/Ast.scala

```diff
@@ -397,6 +397,14 @@ case object Synthesized extends Info {
   lazy val isCached = false
 }
 
+/** An `Info` instance for labelling an AST node which is expected to fail verification.
```

Comment:
If this is intended for Silicon, why not declare it there?

I'm also not a fan of the name `ExpectFail` - doesn't seem to be a noun nor a verb. How about `FailureExpected`? Or `FailureExpectedInfo`, to be in line with, e.g. `ConsInfo`.

Reply:
Will rename to `FailureExpectedInfo`. The issue is that if I declare it in Silicon then I can't use it in the refutation plugin; `case object RefuteInfo extends FailureExpectedInfo`